hocr-pdf

```diff
@@ -73,7 +73,7 @@ def export_pdf(playground, default_dpi, savefile=False):
 
 def add_text_layer(pdf, image, height, dpi):
     """Draw an invisible text layer for OCR data"""
-    p1 = re.compile('bbox((\s+\d+){4})')
+    p1 = re.compile('bbox((\s+[\d\-]+){4})')
```

Comment:
This would not only allow `-1234`, but also `12-3-4` and other illegal patterns. Are negative values allowed and reasonable?